packages/maumau-server/src/controllers/pool.ts

```diff
@@ -44,9 +44,14 @@ export default class PoolController {
     );
   }
 
-  private join = (request: Request<{}, {}, { id: string; name: string }>, response: Response<{ status: Status }>) => {
-    const { id, name } = request.body;
-    this.matchmakerService.joinPool({ id, name });
+  private join = (request: Request<{}, {}, { id: string; name: string, playAgainstBot: boolean | undefined }>, response: Response<{ status: Status }>) => {
```

Comment:
Ich überlege ob man vllt in den periodischen matchmaker etwas einbaut, dass einen Bot automatisch joint, wenn ein Spieler zu lange in der Warteschlange ist... vllt können wir dadurch mehr spieler zum spielen anregen. macht das sinn?

Reply:
Gute Idee. Kannst ja mal ein Ticket anlegen. 👍

Reply:
Jup, habe ich gemacht: https://github.com/emanuelschmitt/maumau/issues/37